src/katas/arg-extract1.clj

```diff
@@ -0,0 +1,21 @@
+(ns zombietrainers.arg-extract1)
+
+(defn arg-extract
+  "Write a function takes a vector as an argument, `arg`
+   extracts every other value and places it into a new vector. "
+  [arg]
+  (->> arg       ;; "->>" is the thread-last macro
+       (cons 0)
+       (take-nth 2)
+       rest
+       vec))
+
+(comment
```

Comment:
So usually it's best practice to delete the comment block.

Reply:
I know Rich Hickey personally, and he approves of my usage.